### ISSUE TITLE
Support for dynamic schemas

### ISSUE THREAD

**Issue by deivkk** [vektah/gqlgen#154]
I am allowing some users to create dynamic fields for a specific entity (`Person`) within the context of another entity (`Project`). `Person` has some common fields, but others are created dinamically per `Project` by these privileged users.

Is there any way to extend such behaviour to GraphQL? In terms of data storage I am working with MongoDB since it offers a lot of flexibility for working on dynamic scenarios, however I don't know how to approach it for GraphQL.

Basically I am considering the possibility of querying fields from `Person` (might be different depending on the `Project`) but also create mutations to edit these fields.

**Comment by vektah** [comment]
Its probably best to ask for help on [gitter](https://gitter.im/gqlgen).

Without knowing much about the structure of your app I cant really suggest much:
 - If the properties fall into nice groups of types you can use interfaces
 - If its truly dynamic you can use a `Map` but doing that will break all the type safety graphql offers you.
